lib/configs/SolutionLibraries/KernelsTileLite.yaml

```diff
@@ -0,0 +1,205 @@
+GlobalParameters:
```

Comment:
The SolutionLibraries directory contains built solution libraries (that can be loaded by the host code).  This could go in 'configs' or in a new subdirectory of 'configs' along with sgemm_all_transposes.yaml.